src/helpers/fallbackauthdialog.ts

```diff
@@ -44,23 +49,18 @@ const msalConfig: Msal.Configuration = {
   },
 };
 
-var requestObj: Object = {
+var requestObj: RedirectRequest = {
```

Comment:
You need to request Graph scopes as "extraScopesToConsent". You don't want the access token to actually have Graph scopes from the client side. The server will request that later through the OBO flow.

Reply:
Can you explain a little more?  I was just changing the msal library being used and keeping the rest of it as it already was.  This sounds like something that needs to be done as part of the other change I'm making to move the token handling to middle-tier . . . but I'm not sure what changes to the scope should look like or why.

Reply:
Here's an example of [how to use it](https://github.com/OfficeDev/Office-Add-in-samples/blob/5668fa059457f218570f4347f67d859383b5736e/Samples/auth/Office-Add-in-NodeJS-SSO/Complete/public/javascripts/fallback-msal/authConfig.js#L14) Would be nice if you could pick up the change here, but if you are covering it in a separate PR that works. Dan recommended this pattern to me so he knows more about the implementation details and the risk.

Reply:
I can do it in either spot (the template won't be enabled until they are both in).

I've been looking at your sample as an example of where I should be making changes and I saw the scope that you had . . . however I'm not sure what I should put for extraScopesToConsent for my case (I'm not reading files, I'm reading user profile information) and I'm not sure if I should be changing the scopes property here as well.

Reply:
I'll look into this in the other change I working on.